metrics/issues-open.md

```diff
@@ -10,7 +10,7 @@ Note: do not confuse this with the opened tickets in a specific period of time r
 
 Mandatory: 
 * Point in time. This is the time for which the snapshot of the open issues is computed.
-* Statuses. These are the statuses considered as open. Platform and project dependant.
+* Statuses. These are the statuses considered as open. This component is platform and project dependent.

```

Comment:
Thanks for noticing the typo with "dependant". For the rest of this change, I think I prefer the terser form. So, please change it to "Platform and project dependent".

Reply:
I have made the required changes. Please have a look.